src/scenes/Login/Login.tsx

```diff
@@ -0,0 +1,102 @@
+import { useState } from "react";
+import Banner from "../../components/Banner";
+import Button from "../../components/Button";
+import { EButtonType } from "../../components/Button/Button";
+import ErrorMessage from "../../components/ErrorMessage";
+import TextField from "../../components/TextField";
+import "./login-style.css";
+
+interface ILoginSceneProps {
+  onDismiss?: () => void;
+}
```

Comment:
For these different "scenes" we might consider adding an optional prop that indicates the scene (EAppScene) from which the current component was triggered so when we dismiss the scene, we can render the component that triggered it.